### PR TITLE
MSTest sample Selenium 4

### PR DESCRIPTION
**Pull request by Baaryan**

### CODE REVIEW

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]
Similar comments as selenium-3

**Comment by Baaryan** [comment]
Done

**Comment by Baaryan** [comment]
Done

File: MSTest-browserstack/mac.conf.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     "browserVersion": "16.5",
     "local": "false",
     "buildName": "browserstack-build-1",
-    "sessionName": "BStack test",
-    "source": "mstest:sample"
+    "name": "BStack parallel mstest",
+    "browserstack.source": "mstest:sample-main:v1.0"
```
Comment (kamal-kaur04) [inline MSTest-browserstack/mac.conf.json]: `source` will be the capability

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

**Review by kamal-kaur04 — APPROVED** [review]

**Review by pranavj1001 — APPROVED** [review]

**Pull request merged**